product_supplierinfo_for_customer/models/product_product.py

```diff
@@ -130,10 +130,22 @@ def _select_customerinfo(
             params = dict()
         params.update({"partner_id": partner.id})
         domain = self._prepare_domain_customerinfo(params)
-        res = (
+        # Search for customerinfo records based on the domain
+        any_match = (
             self.env["product.customerinfo"]
             .search(domain)
             .sorted(lambda s: (s.sequence, s.min_qty, s.price, s.id))
```

Comment:
@AaronHForgeFlow  Why 'sorted' and not the use of the `order_by` parameter when calling the `search` method? This will avoid a useless in memory iteration done by the call to `sorted`

Reply:
good point again :) changed